File: CS/WindowsApplication3/Views/OrderView.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Windows.Forms;
using DXSample.ViewModels;
using DevExpress.XtraEditors.Controls;

namespace DXSample.Views {
    [DevExpress.Utils.MVVM.UI.ViewType("OrderView")]
    public partial class OrderView : UserControl {
        public OrderView() {
            InitializeComponent();
            if (!DesignMode) {
                var fluentApi = mvvmContext.OfType<OrderViewModel>();
                fluentApi.SetObjectDataSourceBinding(bindingSource, x => x.Entities);
            }
            InitImageComboBox();
        }
        private void InitImageComboBox() {
            for (int i = 0; i < imageCollection1.Images.Count; i++) {
                repositoryItemImageComboBox1.Items.Add(new ImageComboBoxItem(i, i));
            }
        }
    }
}

File: VB/WindowsApplication3/Views/OrderView.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Windows.Forms
Imports DXSample.ViewModels
Imports DevExpress.XtraEditors.Controls

Namespace DXSample.Views
    <DevExpress.Utils.MVVM.UI.ViewType("OrderView")> _
    Partial Public Class OrderView
        Inherits UserControl

        Public Sub New()
            InitializeComponent()
            If Not DesignMode Then
                Dim fluentApi = mvvmContext.OfType(Of OrderViewModel)()
                fluentApi.SetObjectDataSourceBinding(bindingSource, Function(x) x.Entities)
            End If
            InitImageComboBox()
        End Sub
        Private Sub InitImageComboBox()
            For i As Integer = 0 To imageCollection1.Images.Count - 1
                repositoryItemImageComboBox1.Items.Add(New ImageComboBoxItem(i, i))
            Next i
        End Sub
    End Class
End Namespace
